src/opm/input/eclipse/EclipseState/Grid/MULTREGTScanner.cpp

```diff
@@ -145,13 +146,13 @@ std::vector<int> unique(const std::vector<int>& data) {
         }
 
         for (auto iter = searchPairs.begin(); iter != searchPairs.end(); ++iter) {
```

Comment:
ranged for?

Reply:
> ranged for?

Good idea.  I've pushed an update to that effect.